dist/pythonlibs/testrunner/__init__.py

```diff
@@ -44,3 +44,13 @@ def run(testfunc, timeout=TIMEOUT, echo=True, traceback=False):
         print("")
         teardown_child(child)
     return 0
+
+
+def check_unittests(timeout=TIMEOUT, echo=True, traceback=False,
+                    nb_tests=None):
+    _tests = '\d+' if nb_tests is None else int(nb_tests)
+
+    def _check_func(child):
+        child.expect(r'OK \({} tests\)'.format(_tests), timeout=timeout)
```

Comment:
I rather would prefer this function also to be available publicly (with the `_tests` variable and timeout parameter added here as well). There are tests that not only run the unittests (e.g. I have written some that first do some unittests and then start the shell for more interactive tests), that then could benefit from that function.